data/eos-updater-flatpak-installer.service.in

```diff
@@ -0,0 +1,41 @@
+[Unit]
+Description=Endless OS Automatic Updater - Post Boot Flatpak Installer
+Wants=local-fs.target
+After=local-fs.target
+ConditionKernelCommandLine=!eos-updater-disable
+
+# Only on updates
+Before=multi-user.target systemd-update-done.service
+ConditionNeedsUpdate=/var
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStart=@libexecdir@/eos-updater-flatpak-installer
+
+# Sandboxing
+# FIXME: Enable more of these options once we have systemd > 229
+Environment=GIO_USE_VFS=local
+Environment=GVFS_DISABLE_FUSE=1
+Environment=GVFS_REMOTE_VOLUME_MONITOR_IGNORE=1
+Environment=GSETTINGS_BACKEND=memory
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateDevices=yes
+PrivateNetwork=no
```

Comment:
`eos-updater-flatpak-installer` shouldn’t have to access the network, should it? The pulls of all the flatpaks are done by `eos-updater`.

Reply:
Yup, that makes sense. I basically cargo-culted this file 😅